base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +419,52 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen;
+    int i;
+
+    if (this->m_dcGrid == NULL)
+        return;
+
+    HPEN solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
```

Comment:
No reason to split `HPEN solidPen` off from the other variables:
all at start is more backward-compatible, all here would work too.